packages/firebase_auth/lib/src/firebase_user.dart

```diff
@@ -40,7 +40,14 @@ class FirebaseUser extends UserInfo {
       'app': _app.name,
     });
 
-    return IdTokenResult(data, _app);
+    if (data != null) {
```

Comment:
Is there a way to get the `data` to be `null` for integration testing? If not, I think a unit test of the changed behavior would be fine.

Reply:
You are right. I don't think there is a way to get null if the Task completed successful. The only reason the IDE suggests a null check here is because of the Task API witch allow for the result of a task to be null. But does not apply in this case.